src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/events/ReferralEventPublisher.kt

```diff
@@ -8,7 +8,7 @@ import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.controller.Referra
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
 
 enum class ReferralEventType {
-  SENT, ASSIGNED,
+  SENT, ASSIGNED, CANCELLED, PREMATURELY_ENDED, COMPLETED
```

Comment:
nice - this fits in well with the existing events

Reply:
Phew!